rajawali/src/main/java/rajawali/renderer/RajawaliRenderer.java

```diff
@@ -1310,4 +1410,63 @@ public void setUsesCoverageAaAll(boolean usesCoverageAa) {
 			}
 		}
 	}
+
+    @SuppressLint("HandlerLeak")
```

Comment:
The suppress may be warranted here as items in the async queue can still be processing after the renderer is stopped as nothing seems to stop the queue. Unless I missed it, the queue of loaders should be nuked and the callbacks released.